client/flutter/lib/pages/symptom_checker/symptom_checker_model.dart

```diff
@@ -38,25 +37,33 @@ class SymptomCheckerModel with ChangeNotifier {
   /// the pages list. If the series is complete the isComplete flag will be set.
   /// In both cases the change notifier will fire to indicate the update.
   void answerQuestion(Set<String> answerIds) {
+    try {
+      isComplete = _answerQuestionImpl(answerIds);
```

Comment:
If there is some error in computing next question with the logic, but it isn't caught by this try/catch, we risk thinking that the user has completed the full survey when they haven't... I can't foresee exactly why that would happen, but just noting it here as a risk

Reply:
not sure how that's possible with Dart's spec, but i set isFatalError to true before the try, and set it to false after isComplete to be absolutely sure.